pkg/consts/consts.go

```diff
@@ -41,5 +41,5 @@ var (
 	NewBaseHashFunc = sha256.New
 
 	// DataCommitmentBlocksLimit is the limit to the number of blocks we can generate a data commitment for.
-	DataCommitmentBlocksLimit = 1000
+	DataCommitmentBlocksLimit = 2048
```

Comment:
How was `2048` derived? I would expect `1200` instead of `2048` because:

4 (hours) * 60 (minutes) * 60 (seconds) = 14,400 seconds in 4 hours
14,400 / 12 (slot time on Ethereum) = 1,200
14,400 / 12 (block time on Celestia) = 1,200

Reply:
To have a small extra wiggle room. I am currently discussing bumping it to 10000, but we're still investigating the impacts with Succinct

Reply:
2048 * 12 = 24,576 / 60 / 60 = 6.8 hours so almost 7 hours. Given this isn't close to 4 hours, can the PR description be updated? 

Also can the rationale for the value chosen be documented in a code comment next to the line if it's not immediately obvious.